contrib/linux/actions/scp.yaml

```diff
@@ -13,10 +13,6 @@
             type: 'string'
             description: 'Destination of files/directories'
```

Comment:
Let's please add an example to the description to make it clear destination can be relative, absolute, or full server:path notation.